employees/templates/employees/project_report_detail.html

```diff
@@ -0,0 +1,26 @@
+{% extends 'base.html' %}
+
+{% load i18n %}
+{% load static %}
+{% load crispy_forms_tags %}
+
+{% block extra_head %}
+<link rel="stylesheet" type="text/css" href="{% static 'employees/style.css' %}">
+{% endblock %}
+
+{% block content %}
+<h1>{{ UI_text.PAGE_TITLE.value }}{{ report.project }} ({{ report.date }})</h1>
+</br>
+<div class="modal-dialog">
+    <form action="{% url 'project-report-detail' pk=report.pk %}" method="POST">
+        {% csrf_token %}
+        {{ form|crispy }}
+        <input type="submit" value="{{ UI_text.UPDATE_REPORT_BUTTON.value }}">
+        <input type="submit" name="discard" value="{{ UI_text.DISCARD_CHANGES_BUTTON.value }}">
```

Comment:
Is this being handled by Django by default? 

Generally I don't like making a POST request just to **not** make any changes. I think from UX perspective it is simpler to "go back" or "cancel" or something like that. I would suggest replacing it with a link back.

Reply:
I'm happy to inform you that everything is in absolute order and this solution in fact does not work at all. 😄 

Not only does the test fail, I also cannot see any other button except for **Update** in rendered view. I guess I must have skipped the unit tests on this branch, during the hassle of multiple rebases.

Reply:
Well, I'm happy to inform you that it does not work at all. 😁 

The _discard_ button just works as another submit, changes apply. I'll fix it up in a jiff.

Reply:
Done